user_sync/app.py

```diff
@@ -371,28 +371,46 @@ def credentials():
     help="Stores all sensitive fields and updates configuration files, replacing plaintext values with keys")
 def store():
     """
-    Stores credentials in the configuration file
+    Stores secure credentials in the configuration file
     This is an automated process
     """
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    click.echo("You have called the store-credential command.")
+    credential_manager.store()
+    retrieve_config_credentials = credential_manager.retrieve_config_credentials()
+    for cred_identifier in retrieve_config_credentials:
+        click.echo(f"{cred_identifier} stored securely in configuration file.")
 
 
-@credentials.command(help="Retreives currently stored credentials under the username 'user_sync'.")
-@click.option('--revert', default=False, is_flag=True, help="Reverts configuration files to plaintext state.")
-def retrieve(revert):
+@credentials.command(help="Retrieves currently stored credentials under the username 'user_sync'.")
+def retrieve():
     """
     Retrieves credentials from the configuration files
     By default, just prints out the values stored for UST
-    If --revert is used, updates config files with actual plaintext data
-    This is an automated process
     """
     credential_manager = CredentialManager()
-    click.echo("Using backend: " + credential_manager.keyring_name)
-    if revert:
-        click.echo("reverting...")
-    click.echo("you have called the retrieve credential command")
+    retrieve_from_credman_keys = credential_manager.retrieve_from_credman_keys()
+
+    if retrieve_from_credman_keys is None:
+        raise AssertionException("Unable to retrieve credentials from configuration files.")
+    for key_value in retrieve_from_credman_keys.items():
+        click.echo(f"The following values were stored securely: {key_value}")
+
+
+@credentials.command(help="Will return configuration file to unsecured state and replace all secrure values with "
+                          "plain text values.")
+def revert():
+    """
+    Revert updates config files
+    with actual plaintext data This is an
+    automated process.
+    """
+    credential_manager = CredentialManager()
+    credential_manager.revert()
+    retrieve_config_credentials = credential_manager.retrieve_config_credentials()
+    for plaintext_cred_identifier in retrieve_config_credentials:
```

Comment:
try this  - shorten the method names and combine:

for identifier in credential_manager.revert():

revert can return a list of the keys it changed

Reply:
can we change the method names in credential manager to be more concise? Eg - just revert, retrieve, etc.  No need for the _config_credentials  or _from_credman_keys - the fact that these methods are part of the class keep the namespace clear.